README.md

```diff
@@ -1,4 +1,14 @@
-# mlibc is a C standard library
+# mlibc
+
+mlibc is a fully featured C standard library designed with portability in mind.
+
+We support a number of architectures (x86-64, i386, RISC-V, AArch64, m68k), and provide a clean syscall abstraction layer for new operating system ports to plug into.
+
+Unlike other portable C standard libraries like newlib, we aim for feature parity with glibc/musl i.e full pthread support and other GNU extensions.
+
+mlibc is [capable enough to run web browsers](https://forum.osdev.org/viewtopic.php?p=345952#p345952), though some features are still missing.
```

Comment:
Not sure if that's a good achievement or not (do web browser require a lot of libc functionality?). If anything, I'd list more than one example here. Otherwise, LGTM.

Reply:
@Dennisbonke what else can we mention here?